addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,42 +1,75 @@
 $gold: #eca801;
 $silver: #cccccc;
 $bronze: #eea91e;
+$o-wf-main-wrapper-width: map-get($container-max-widths, xl);
+$o-wf-main-wrapper-narrow-width: map-get($container-max-widths, md);
+$o-wf-sidebar-width: 16em !important;
+
 
 .website_forum {
-    .o_forum_ask_btn {
-        @include media-breakpoint-up(md) {
-            box-shadow: $box-shadow;
-            width: 200px;
-        }
-    }
+    --o-main-wrapper-width: #{$o-wf-main-wrapper-narrow-width};
+    --o-sidebar-width: #{$o-wf-sidebar-width};
 
-    .o_wforum_nav .nav-link {
-        @include o-hover-text-color($body-color, $link-color);
-        line-height: 1;
+    @include media-breakpoint-up(md){
+        --o-main-wrapper-width: #{$o-wf-main-wrapper-width};
+    }
 
-        .fa {
-            opacity: 0.5;
+    .o_wforum_main_wrapper {
+        @include media-breakpoint-up(md){
+            grid-template-columns: 1fr minmax(0, var(--o-main-wrapper-width)) 1fr;
+            gap: 1.2rem;
```

Comment:
mmm... the sidebar is calculated in `em`, the wrapper in `px`, the gap in `rem` .
Let's triple check that all these different values work well together.
I'm mostly concerned about the sidebar's width that's relative to the font-size... what happens if the font-size is very big?